dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/TracesFeature.kt

```diff
@@ -58,7 +58,7 @@ internal object TracesFeature {
 
         clientToken = config.clientToken
         endpointUrl = config.endpointUrl
-        serviceName = config.serviceName
+        serviceName = config.serviceName ?: appContext.packageName
```

Comment:
Instead of copy/pasting this logic everywhere, you could define it once and for all in a `CoreFeature.serviceName` property, that'll be then used by other features ?

Reply:
Was actually thinking the same...but CoreFeature doesn't hold a reference to the FeatureConfig or RumConfig. I would have to actually provide the FeatureConfig as a parameter to that method and because  RumFeature has a different config (RumConfig) I will need 2 methods. I am not really sure which approach will be better: having the code duplicated in each feature or having a 2 methods in the CoreFeature :(. Maybe I can think to something else.